### PR TITLE
WIP: Guideline to create a new protocol binding

### PR DESCRIPTION
**Pull request by sebastiankb**
also see #103

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-binding-templates/pull/104.html" title="Last updated on Mar 5, 2021, 1:58 PM UTC (900c124)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-binding-templates/104/b524968...900c124.html" title="Last updated on Mar 5, 2021, 1:58 PM UTC (900c124)">Diff</a>

### CODE REVIEW

**Review by egekorkan — APPROVED** [review]

**Comment by egekorkan** [comment]
Looks good to me, can be merged in the call tomorrow.

File: index.html [hunk 1]
```diff
@@ -1216,6 +1216,29 @@ <h2>Bindings for Events</h2>
...
+      <h1>Guideline to write new Protocol Binding</h1>
+      <p>To provide a new binding specification, e.g. based on a new communication protocol, a 
+        minimal requirements for information are required, which can then be used for distinct 
+        implementation and to ensure interoperability based on the new binding.
+      <p>
+        <!--be associated with WoT's <a>Properties</a>,
+      <a>Actions</a>, and <a>Events</a> the following minimum of requirment of information is required:</p>
+        -->
+      <p>
+        Please ensure that following information are provided and common shared:
```
Comment (relu91) [inline index.html:1232]: *Please ensure that following information are provided and common shared* -> *Please ensure that the following information is provided and commonly shared*

File: index.html [hunk 2]
```diff
@@ -1216,6 +1216,29 @@ <h2>Bindings for Events</h2>
         </p>
       </section>
 
+
+
+    </section>
+    <section>
+      <h1>Guideline to write new Protocol Binding</h1>
+      <p>To provide a new binding specification, e.g. based on a new communication protocol, a 
+        minimal requirements for information are required, which can then be used for distinct
```
Comment (relu91) [inline index.html:1225]: a minimal -> minimal

File: index.html [hunk 3]
```diff
@@ -1216,6 +1216,29 @@ <h2>Bindings for Events</h2>
...
+      <a>Actions</a>, and <a>Events</a> the following minimum of requirment of information is required:</p>
+        -->
+      <p>
+        Please ensure that following information are provided and common shared:
+        <li>For identification of the used protocol a standardized protocol scheme value has to be used in the URI scheme of top level <code>base</code> or <code>href</code> term of the <code>forms</code> container (also see [[RFC3986]]). E.g., <code>"https://..."</code>, <code>"mqtt://..."</code>, <code>"modbus://..."</code></li>  
+        <li>Depending of the complexity of the protocol a corresponding and established context may be introduced within the <code>@context</code> to decleare all needed protocol features by corresponding vocabulary terms (e.g., <code>baudrate</code>). </li>
+        <li>
+          A protocol binding MUST be able to map WoT operation types (readproperty, invokeaction, etc.) for the <code>op</code> term to concrete protocol methods. 
+          This includes that the <code>href</code> term of the <code>forms</code> container have to be used to encode the address information of the protocol for the
```
Comment (relu91) [inline index.html:1237]: have -> has

File: index.html [hunk 4]
```diff
@@ -1216,6 +1216,29 @@ <h2>Bindings for Events</h2>
...
+        minimal requirements for information are required, which can then be used for distinct 
+        implementation and to ensure interoperability based on the new binding.
+      <p>
+        <!--be associated with WoT's <a>Properties</a>,
+      <a>Actions</a>, and <a>Events</a> the following minimum of requirment of information is required:</p>
+        -->
+      <p>
+        Please ensure that following information are provided and common shared:
+        <li>For identification of the used protocol a standardized protocol scheme value has to be used in the URI scheme of top level <code>base</code> or <code>href</code> term of the <code>forms</code> container (also see [[RFC3986]]). E.g., <code>"https://..."</code>, <code>"mqtt://..."</code>, <code>"modbus://..."</code></li>  
+        <li>Depending of the complexity of the protocol a corresponding and established context may be introduced within the <code>@context</code> to decleare all needed protocol features by corresponding vocabulary terms (e.g., <code>baudrate</code>). </li>
```
Comment (relu91) [inline index.html:1234]: decleare -> declare

**Review by relu91 — CHANGES_REQUESTED** [review]
Just some minor typos. Other than that it looks ok!

**Review by OhArthits — APPROVED** [review]

**Pull request closed without merging**